src/native/fs/fs_napi.cpp

```diff
@@ -268,6 +280,24 @@ set_fs_worker_stats(Napi::Env env, Napi::Object fs_worker_stats, std::string wor
     fs_worker_stats["error"] = Napi::Number::New(env, error);
 }
 
+int
+cmp_ver_id(int link_expected_mtime_sec, int link_expected_mtime_nsec, double link_expected_inode, struct stat& _stat_res)
```

Comment:
I think it's better to keep using int64 for both sec/nsec and not assume it will always fit into int.